beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -2267,22 +2083,37 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         })
     }
 
-    /// Attempt to verify and import a chain of blocks to `self`.
-    ///
-    /// The provided blocks _must_ each reference the previous block via `block.parent_root` (i.e.,
-    /// be a chain). An error will be returned if this is not the case.
-    ///
-    /// This operation is not atomic; if one of the blocks in the chain is invalid then some prior
-    /// blocks might be imported.
+    /// A convenience method for spawning a blocking task. It maps an `Option` and
+    /// `tokio::JoinError` into a single `BeaconChainError`.
+    pub(crate) async fn spawn_blocking_handle<F, R>(
+        &self,
+        task: F,
+        name: &'static str,
+    ) -> Result<R, Error>
+    where
+        F: FnOnce() -> R + Send + 'static,
+        R: Send + 'static,
+    {
+        let handle = self
+            .task_executor
+            .clone()
```

Comment:
this `clone()` seems unnecessary and the code compiles without it

Reply:
Nice one, addressed in 65c1da107.